Makefile

```diff
@@ -1078,8 +1078,18 @@ test-debug:
 	env LIBSTORAGE_DEBUG=true $(MAKE) test
 
 test-rbd:
-	DRIVERS=rbd $(MAKE) deps
-	DRIVERS=rbd $(MAKE) ./drivers/storage/rbd/tests/rbd.test
+	env DRIVERS=rbd $(MAKE) deps
```

Comment:
my inspiration here was the `test-debug` target directly above. What is it about its scenario that makes `env` desirable and the driver-specific test builds not?